scripts/check_job_periodicity.py

```diff
@@ -186,7 +190,8 @@ def do_job_query(url, job_type, job_status):
         "sort": [{"job.job_info.time_end": {"order": "desc"}}],
         "_source": ["job_id", "payload_id", "payload_hash", "uuid",
                     "job.job_info.time_queued", "job.job_info.time_start",
-                    "job.job_info.time_end", "error", "traceback"],
+                    "job.job_info.time_end", "job.job_info.time_limit",
+                    "error", "traceback"],
         "size": 1
     }
     logging.info("query: %s" % json.dumps(query, indent=2, sort_keys=True))
```

Comment:
@mkarim2017: while you're updating these scripts, could you also update the way we query ES in them by utilizing the common ES library. Essentially we're getting rid of the use of `requests` to query ES as we encounter them throughout HySDS core repos. For example, https://github.com/hysds/hysds/blob/30611dced6f7788a6915b16344276a238c575f86/hysds/event_processors.py#L84.

Reply:
Updated for ES query. Kept for the slack put.